ocs_ci/ocs/resources/pod.py

```diff
@@ -212,7 +215,61 @@ def copy_to_pod(self, src_path, target_path, container=None):
             cmd = cmd + f" -c {container}"
         return self.ocp.exec_oc_cmd(cmd, out_yaml_format=False)
 
-    def exec_sh_cmd_on_pod(self, command, sh="bash", timeout=600):
+    def copy_to_pod_cat(self, src_path, target_path, timeout=60):
+        """
+        Copies to pod path from the local path file using standard output stream
+
+        Args:
+            src_path (str): local path
+            target_path (str): path within pod where you want to copy
+            timeout (int): timeout in seconds
+        Returns:
+            str: stdout of the command
+        """
+        cmd = f"cat {src_path} | oc exec -i {self.name} -n {self.namespace} -- sh -c 'cat > {target_path}'"
+        return exec_cmd(cmd, timeout=timeout, use_shell=True)
+
+    def copy_from_pod_oc_exec(
+        self, target_path, src_path, timeout=600, chunk_size=2000
+    ):
+        """
+        Copies to local path file from the pod using standard output stream via 'oc exec'. Good for log/json/yaml/text
+        files, not good for large files/binaries with one-line plain string
+        * Hand data from the pod over `oc exec cat`, other standard output ways will fail for the files > 1 Mb
+
+        Args:
+            target_path (str): local path
+            src_path (str): path within pod what you want to copy
+            timeout (int): timeout in seconds
+                until size of the local file will not reach the initial file size.
+                2000 lines is a maximum chunk size tested successfully
+            chunk_size (int): file will be copied by chunks, by number of lines
+        """
+        file_size = int(self.exec_cmd_on_pod(f"stat -c %s {src_path}"))
+
+        timestamp_end = datetime.now() + timedelta(seconds=timeout)
+
+        x = 1
```

Comment:
can we give some meaningful var name?

Reply:
Sorry for misunderstanding. Curren PR was in WIP. I am closing it. Separated my changes from side commits to https://github.com/red-hat-storage/ocs-ci/pull/7178